cli/src/main/java/org/jboss/as/cli/handlers/Utils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.cli.handlers;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+/**
+ * Safe close utility class.
+ *
+ * @author <A href="mailto:ropalka@redhat.com">Richard Opalka</A>
+ */
+public final class Utils {
+
+    private Utils() {
+        // forbidden instantiation
+    }
+
+    public static void safeClose(final Closeable closeable) {
```

Comment:
this could be probably eliminated by using try with resources in code that now uses it.